test-files/cmd/glacial/NonDestructiveConflationTest.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+mkdir -p test-output/NonDestructiveTest/
+
+# First run the Congo network conflation
+hoot conflate -C core/Network.conf -D reader.conflate.use.data.source.ids.1=true \
+ test-files/Congo_MGCP_Roads_Bridges_subset.osm \
+ test-files/Congo_OSM_Roads_Bridges_subset.osm \
+ test-output/NonDestructiveTest/output.osm
+
+
+# Derive the changeset for the Congo conflation
+hoot derive-changeset --stats \
+ test-files/Congo_MGCP_Roads_Bridges_subset.osm \
+ test-output/NonDestructiveTest/output.osm \
+ test-output/NonDestructiveTest/changeset.osc
```

Comment:
Simple test, use Congo roads bridges with network conflation and then produce the stats of the changeset.